projects/igniteui-angular/src/lib/grids/hierarchical-grid/row-island.component.ts

```diff
@@ -293,7 +293,6 @@ export class IgxRowIslandComponent extends IgxHierarchicalGridBaseDirective
 
         // handle column changes so that they are passed to child grid instances when onColumnChange is emitted.
         this.ri_columnListDiffer.diff(this.childColumns);
-        this.childColumns.toArray().forEach(x => x.onColumnChange.pipe(takeUntil(x.destroy$)).subscribe(() => this.updateColumnList()));
```

Comment:
This handles changes from the row island column properties and applies them to already existing child grids so that the change is reflected in them. Removing it will probably break that scenario. For example, if a child grid is already expanded and a property on the related row island column is changed (for example, `header` prop is changed) after this, this change will never be reflected on this existing child grid.